src/main/java/org/springframework/cloud/deployer/spi/kubernetes/KubernetesScheduler.java

```diff
@@ -204,15 +206,23 @@ protected CronJob createCronJob(ScheduleRequest scheduleRequest) {
 				schedulerProperties.get("spring.cloud.deployer.kubernetes.cron.expression") :
 				schedulerProperties.get(SchedulerPropertyKeys.CRON_EXPRESSION);
 		Assert.hasText(schedule, "The property spring.cloud.deployer.cron.expression must be defined");
-		
+
 		String concurrencyPolicy = schedulerProperties.get(KUBERNETES_DEPLOYER_CRON_CONCURRENCY_POLICY);
 		// check default server properties
 		if (!StringUtils.hasText(concurrencyPolicy)) {
 			concurrencyPolicy = this.properties.getCron().getConcurrencyPolicy();
 		}
 		if(concurrencyPolicy==null) {
 			concurrencyPolicy = "Allow";
-		} 
+		}
+
+		final Integer ttlSecondsAfterFinished;
+		String ttlSecondsAfterFinishedString = schedulerProperties.get(KUBERNETES_DEPLOYER_CRON_TTL_SECONDS_AFTER_FINISHED);
+		if (StringUtils.hasText(ttlSecondsAfterFinishedString)) {
+			ttlSecondsAfterFinished = Integer.parseInt(ttlSecondsAfterFinishedString);
+		} else {
+			ttlSecondsAfterFinished = this.properties.getCron().getTtlSecondsAfterFinished();
```

Comment:
Did you forget to add a file? ;) 

https://github.com/spring-cloud/spring-cloud-deployer-kubernetes/actions/runs/4264049330/jobs/7422743930#step:6:3054